Blockzilla/AppDelegate.swift

```diff
@@ -50,10 +50,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate, ModalDelegate, AppSplashC
         setupErrorTracking()
         setupTelemetry()
         TPStatsBlocklistChecker.shared.startup()
+        browserViewController.tipManager = TipManager()
 
         // Count number of app launches for requesting a review
         let currentLaunchCount = UserDefaults.standard.integer(forKey: UIConstants.strings.userDefaultsLaunchCountKey)
         UserDefaults.standard.set(currentLaunchCount + 1, forKey: UIConstants.strings.userDefaultsLaunchCountKey)
+        
+        // Set original default values for showing tips
+        let tipDefaults = [TipManager.TipKey.autocompleteTip: true,
```

Comment:
Does this reset the tips each app load?

Reply:
Nope, this is just the default value if the value has never been set before.